src/config/bindings.rs

```diff
@@ -177,9 +177,11 @@ pub fn default_key_bindings() -> Vec<KeyBinding> {
     bindings
 }
 
+#[allow(clippy::let_and_return)]
```

Comment:
I had to add this override (and the `unused_mut` one below) because the `bindings` vec is only mutated on Windows. Open to suggestions on better ways to do this.